src/hotspot/share/precompiled/precompiled.hpp

```diff
@@ -25,53 +25,27 @@
 // Precompiled headers are turned off if the user passes
 // --disable-precompiled-headers to configure.
 
-// These header files are included in at least 130 C++ files, as of
-// measurements made in November 2018. This list excludes files named
-// *.inline.hpp, since including them decreased build performance.
+// These header files are selected using the output of Clang
+// '-ftime-trace' as a measure of how much time we spend
+// compiling them.
 
-#include "classfile/classLoaderData.hpp"
-#include "classfile/javaClasses.hpp"
-#include "classfile/systemDictionary.hpp"
-#include "gc/shared/collectedHeap.hpp"
-#include "gc/shared/gcCause.hpp"
-#include "logging/log.hpp"
+#include "classfile/javaClasses.inline.hpp"
+#if INCLUDE_SHENANDOAHGC
```

Comment:
So the common style is to move conditional includes at the end of the include block. I think the same rule applies here.

Reply:
Right, ba6f372d87765417f1bf05cd2b1a7e058c18350d